pima_server.py

```diff
@@ -323,6 +329,7 @@ def ParseArguments() -> argparse.Namespace:
         time.sleep(5)
       else:
         break
+    mqtt_publish_lwt_online()
```

Comment:
Should this be before loop_start?

Reply:
Good question.  I was thinking about that too.
I called the publish method before the loop_start because I didn't want HA to miss the first status message if it comes quickly.  I'm not sure if this is a good enough reason.  You know the code better so you should decide.  Thanks.

Reply:
Seems that it doesn't really matter. This currently means that the online message will be sent on the first loop() call, so it's probably better, though likely nothing else will be there at that point.